src/components/common/helpers/time.ts

```diff
@@ -35,3 +35,8 @@ export function handleUTCTime(ts: string, isRelativeTime = false) {
     }
     return timestamp;
 }
+
+export const formatDurationDiff = (startedOn: string, finishedOn: string) => {
+   const diff : moment.Duration = moment.duration(moment(finishedOn).diff(moment(startedOn)))
```

Comment:
Rename it more precisely

Reply:
we've kept it in helper so name is given accordingly :)